libraries/glew/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# CLONE PHASE
+
+# git clone https://github.com/nigels-com/glew.git glew
+# pushd glew
+# git checkout -f 062067f
+# popd
+
+# The git clone currently gives an error during make, using zip until fixed
```

Comment:
What's the error when trying to build using the git clone?

Reply:
`make: *** No rule to make target 'src/glew.c', needed by 'tmp/linux/default/shared/glew.o'.  Stop.`